### ISSUE TITLE
FinishRelease pushes all branches

### ISSUE THREAD

**Issue by olivierboudet** [RobWin/jgitflow-gradle-plugin#33]
When running finishRelease with pushRelease=true, all branches are pushed (including old ones, with no modifications).

When using Jenkins configured to trigger builds on branch modifications, a build for each branch will be triggered after a finishRelease tasks, which is useless.

Instead of pushing everything to the remote, should not the finishRelease task only push develop, master and the deleted ones ?

**Comment by chenjianjx** [comment]
Yes, this is a problem.  It makes remote-deleted branches come back
